src/App.tsx

```diff
@@ -21,14 +51,13 @@ export const App: React.FC = () => {
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos !== null ? <TodoList todos={todos} /> : <Loader />}
             </div>
           </div>
         </div>
       </div>
 
-      <TodoModal />
+      {showModal && <TodoModal currentUserId={currentUser} todos={todos} />}
```

Comment:
```suggestion
      {!!showModal && <TodoModal currentUserId={currentUser} todos={todos} />}
```

Reply:
showModal is alredy boolean by default

Reply:
Now - yes. But you don`t know how this condition will update in the future.